### PR TITLE
[ADD] eCommerce: checkout process

### PR DESCRIPTION
**Pull request by toaa-odoo**
Adding new page on checkout process for eCommerce. Forward to master.

taskid: 3110554

### CODE REVIEW

File: content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst
Comment (Donapi) [inline content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst:55]:
```suggestion
You can request :guilabel:`Extra Info` from the customer by adding an **extra step** between the
:guilabel:`Address` and :guilabel:`Confirm Order` steps. To do so, go to :menuselection:`Edit
--> Customize` from any checkout page, and enable : :guilabel:`Extra Step Option`
The :guilabel:`Extra Info` step is an online form linked to the quotation or sales order of the customer. The information added in the extra step is available on the quotation or sales order of the customer from the back end, in the **Sales** app.
```

File: content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst
Comment (Donapi) [inline content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst:64]: I would personally prefer to show what's on screen than the customization part
image

File: content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst
Comment (Donapi) [inline content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst:69]:
```suggestion
Terms and conditions
```

File: content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst
Comment (Donapi) [inline content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst:66]:
```suggestion
Confirm order
```

File: content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst
Comment (Donapi) [inline content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst:79]: I would resize the image that looks very big.

File: content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst
Comment (Donapi) [inline content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst:88]:
```suggestion
You can use **building blocks** to add content at any step of the checkout process. To do so, from any **checkout page**, go to :menuselection:`Edit -->
```

Comment (Donapi) [reply content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst:88]: I would not mention "incuding the order confirmation page" as you state just before "at any step.

File: content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst
Comment (Donapi) [inline content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst:99]:
```suggestion
any product's page by going :menuselection:`Edit --> Customize --> Cart`.
```

**Review by Donapi — APPROVED** [review]
Great page, I enjoyed discovering it :) Just a few comments. 
I already approve the PR to speed up the process. Don't forget to ping @xpl-odoo or @jcs-odoo when it's ready for final review.

File: content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst
Comment (toaa-odoo) [reply content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst]: I did add a picture to make it more understandable where that extra page can be found in the process, but not the steps. I don't think it is necessary.

File: content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst
Comment (toaa-odoo) [reply content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst:67]: Good input

File: content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst
Comment (toaa-odoo) [reply content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst]: You have to be on the "Address" step, when you're filing the address. I added that info.

**Review by xpl-odoo — CHANGES_REQUESTED** [review]
Mostly suggestions as usual :)

File: content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst
Comment (toaa-odoo) [reply content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst:58]: Rewrote it to be clearer

**Review by xpl-odoo — APPROVED** [review]
A couple of minor comments, already approved :)

**Comment by toaa-odoo** [comment]
Do not merge. Part of mega push.

**Comment by toaa-odoo** [comment]
@odoo/doc-review I'm aware of the conflict, I'll fix it before merging. Could you ignore that and have a look at the rest of the PR? Thanks!

**Review by AntoineVDV — APPROVED** [review]
@robodoo delegate+

**Comment by toaa-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
